packages/jest-cli/src/reporters/Status.js

```diff
@@ -137,7 +137,7 @@ export default class Status {
 
     // $FlowFixMe
     const width: number = process.stdout.columns;
-    let content = '\n';
+    let content = process.argv.includes('--verbose') ? '\n' : '';
```

Comment:
you should have access to `globalConfig` in here, which you can check for the `verbose` flag